randomizer.py

```diff
@@ -1485,13 +1486,50 @@ def RandomizeDominion(setNames=None, options=None):
             counter += 1
 
         # Get final list of landscape cards
+        landscapeList = random.sample(waySet, len(waySet))[:1]
+        """Figure out card for Way of the Mouse. This uses similar rules to Young
+        Witch, so select a card from the Bane Cards. The card chosen for Way of the
+        Mouse should not be used when determining most additional card rules, but
+        we do need to check if it is a Liaison for ALly inclusion."""
```

Comment:
Triple-quotes should be reserved for docstrings and should not be used for standard inline comments